doc_src/tutorial.rst

```diff
@@ -750,6 +750,7 @@ Now in another shell::
     > echo $EDITOR
     vim
 
+Best practices for using universal variables include `avoiding <https://fishshell.com/docs/current/language.html#universal-variables>`__ directly manipulating ``config.fish`` and ```.config/fish/fish_variables``, and preferring :doc:`funcsave <cmds/fish_add_path>`.
```

Comment:
As it stands, this goes a bit deep immediately for the tutorial - which is supposed to be a very quick overview (and tbh could do with a bit of a rewrite).

How about this:

1. Link to the language docs on it ("See more on universal variables here")
2. Explain: "Universal variables only need to be set once, you don't need to put them in config.fish."
3. Leave off fish_add_path, funcsave (that's probably a mistake?), fish_variables

Reply:
Also don't link to the documentation via the URL - if this is installed locally it will go to the web, and this would *always* link to the current version of documentation. Link like

```
:ref:`link text here <universal-variables>`
```

instead.

Reply:
Agreed. WDYT?